napalm_yang/parsers/text.py

```diff
@@ -30,6 +31,13 @@ def _parse_list_block(cls, mapping):
                 else:
                     key = extra_vars.pop("key")
 
+                if post_process_filter:
+                    from napalm_yang.helpers import template
```

Comment:
I think the best solution would be to move `get_parser ` to here then:

https://github.com/napalm-automation/napalm-yang/blob/dbarrosop/include/napalm_yang/parsers/__init__.py